playbooks/aws/openshift-cluster/provision.yml

```diff
@@ -0,0 +1,264 @@
+---
```

Comment:
I'm not sure this makes sense as a role.  I think that running a playbook that consists of the aws building blocks makes a bit more sense to me.